### PR TITLE
Little improvement

### PR DESCRIPTION
**Pull request by TomRobert**

### CODE REVIEW

**Comment by timzatko** [comment]
Please also increase version to `1.41` as I released `1.40` a few moments ago.

File: TW-Calc.user.js [hunk 1]
```diff
@@ -1516,7 +1495,7 @@ window.TWCalc_inject = function () {
                     
                     TW_Calc.Craft.window.progressBar = new west.gui.Progressbar()
                         .setValue(Character.professionSkill)
-                        .setMaxValue(850);
+                        .setMaxValue(826);
```
Comment (timzatko) [inline TW-Calc.user.js]: Why the new maximum value is 826?

Comment (TomRobert) [reply TW-Calc.user.js]: The recipes currently only go to 825. So the reachable maximum ist 826 right now.

Comment (timzatko) [reply TW-Calc.user.js]: image

The in-game window show 850 as a maximum. I would like to keep it the same as it is in the original game window.

Comment (TomRobert) [reply TW-Calc.user.js]: I know, I just thought, we could fix Inno's mistakes
But alright, I re-changed it.

**Review by timzatko — APPROVED** [review]

**Comment by timzatko** [comment]
Approved, thanks for these improvements!

**Pull request merged**